curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/616965351e74d4689eb6de30.md

```diff
@@ -30,7 +30,7 @@ Your new `meta` element should have a `name` attribute set to `viewport`, and a
 
 ```js
 const meta = [...document.querySelectorAll('meta')];
-const target = meta?.find(m => m?.getAttribute('name') === 'viewport' && m?.getAttribute('content') === 'width=device-width, initial-scale=1.0');
+const target = meta?.find(m => m?.getAttribute('name') === 'viewport' && / *width *= *device-width *, *initial-scale *= *1\.0 */.test(m?.getAttribute('content')));
```

Comment:
It might be better to trim instead of introducing a regex

Reply:
Hmmm, well, for this particular case I don't think so, besides it may add more complexity to the code.

This fix also works for other valid cases like:
`"width = device-width , initial-scale=1.0 "`

PD: But with your comment, I realized that I can improve the solution by instantiating the regex outside the `find`, so that I don't create it in each iteration. 😅

Reply:
Ah, I missed the internal white space